model/policy.go

```diff
@@ -102,17 +117,18 @@ func (model Model) AddPolicy(sec string, ptype string, rule []string) bool {
 }
 
 // AddPolicies adds policy rules to the model.
-func (model Model) AddPolicies(sec string, ptype string, rules [][]string) bool {
-	for i := 0 ; i < len(rules) ; i++{
+func (model Model) AddPolicies(sec string, ptype string, rules [][]string) (bool, [][]string) {
+	for i := 0; i < len(rules); i++ {
 		if model.HasPolicy(sec, ptype, rules[i]) {
-			return false
+			return false, nil
```

Comment:
I believe here we'd better return back an empty slice, I haven't find a `nil` check from caller, am I missing something?

Reply:
Returns `nil` will convert zero slice, it's working.